packages/ui/src/components/Dialog/Dialog.tsx

```diff
@@ -124,52 +114,49 @@ DialogClose.displayName = DialogPrimitive.Close.displayName;
 
 const DialogHeader = ({
   className,
-  size,
+  size = "md",
```

Comment:
The previous code had "20" to pass down as icon Size, that would be `lg`, if this `md` is intended then I'm ok with it :D

Reply:
I changed the default of icon to be 18 instead of 20, that's why is `md` now